<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>get_tweets</class>
 <widget class="QMainWindow" name="get_tweets">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>810</width>
    <height>483</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <weight>50</weight>
    <bold>false</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>10</y>
      <width>201</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Twitter NLP Utilities</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QTextBrowser" name="tweetDisplay">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>751</width>
      <height>221</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="statusDisplay">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>350</y>
      <width>751</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>50</y>
      <width>431</width>
      <height>25</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label_2">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>User handler:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="user">
       <property name="cursor">
        <cursorShape>IBeamCursor</cursorShape>
       </property>
       <property name="maxLength">
        <number>24</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Tweet count:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QDoubleSpinBox" name="tweetCount">
       <property name="decimals">
        <number>0</number>
       </property>
       <property name="minimum">
        <double>10.000000000000000</double>
       </property>
       <property name="maximum">
        <double>200.000000000000000</double>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="getTweets">
       <property name="text">
        <string>Get Tweets</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
